api-reference/ingestion/lease/find-leases-for-tenant.mdx

```diff
@@ -0,0 +1,219 @@
+    </ResponseField>
+
+    <ResponseField name="minimum_days_notice" type="integer" default="none">
+      Minimum days of notice that must be given to break the retrieved lease agreement.
+    </ResponseField>
+
+    <ResponseField name="deposit" type="float" default="none">
+      The deposit amount for the retrieved lease.
+    </ResponseField>
+
+    <ResponseField name="rent" type="float" default="none">
+      The rent amount for the retrieved lease.
+    </ResponseField>
+
+    <ResponseField name="unit_number" type="string" default="none">
+      The unit number for the lease's address.
+    </ResponseField>
+
+    <ResponseField name="address" type="object" default="none">
+      The lease's address.
+      <Expandable title="address">
+        <ResponseField name="building_name" type="string">
+          The name of the building.
+        </ResponseField>
+        <ResponseField name="address_line_1" type="string">
+          The first line of the lease's address.
+        </ResponseField>
+        <ResponseField name="address_line_2" type="string">
+          The second line of the lease's address.
+        </ResponseField>
+        <ResponseField name="city" type="string">
+          The city of the lease's address.
+        </ResponseField>
+        <ResponseField name="country" type="string">
+          The country of the lease's address.
+        </ResponseField>
+        <ResponseField name="state" type="string">
+          The state in which the lease's address is located.
+        </ResponseField>
+        <ResponseField name="postal_code" type="string">
+          The postal code of the lease's address.
+        </ResponseField>
+        <ResponseField name="latitude" type="float">
+          The latitude of the lease's address.
+        </ResponseField>
+        <ResponseField name="longitude" type="float">
+          The longitude of the lease's address.
+        </ResponseField>
+      </Expandable>
+    </ResponseField>
+  </Expandable>
+</ResponseField>
+
+### Errors
+
+Listed below are common errors that may be returned by the endpoint, along with their corresponding status code.
+
+Status Code - 400
+```json Missing Authorization header
+{
+   "error": {
+    "type": "missing_authorization",
+    "message": "Your request does not include an 'Authorization' header with a bearer token for your account."
+  }
+}
+```
+
+Status Code - 401
+```json Invalid/Expired Authorization header bearer token value
+{
+   "error": {
+    "type": "invalid_authorization",
+    "message": "The bearer token you have provided in the 'Authorization' header is invalid."
+  }
+}
+```
+
+Status Code - 400
+
+Returned if none of the tenant parameters are provided in the request body
+```json No tenant information
+{
+   "error": {
+    "type": "no_tenant_information",
+    "message": "You need to provide at least one of identifier, email_address or phone_number in the request body."
+  }
+}
```

Comment:
Should there be two status 400s? If yes can we have them together?

Reply:
By together I meant closer to each other so that the same error code is in one place, and thanks for explaining